### PR TITLE
Adding code to reproduce predictions

### PR DESCRIPTION
**Pull request by angelazhu2**
Adding ozel's branch to reproduce results from the original paper.

### CODE REVIEW

File: reproduce.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    "repr"  # This will be the name of the directory where results for this run are saved.  # this is a subdirectory in save_base
+)
+
+
+"""
+species_set
+- Which set of species to train on.
+- Valid values: 'all', 'snt_birds'
+"""
+train_params["species_set"] = "all"  # unused parameter
```
Comment (angelazhu2) [inline reproduce.py]: planning on deleting the unused parameters

File: reproduce.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+train_params["input_enc"] = "sin_cos"  # unused parameter
+
+"""
+loss
+- Which loss to use for training.
+- Valid values: 'an_full', 'an_slds', 'an_ssdl', 'an_full_me', 'an_slds_me', 'an_ssdl_me'
+"""
+train_params["loss"] = "an_full"  # unused parameter
+
+# train:
```
Comment (angelazhu2) [inline reproduce.py]: was planning on cleaning up the comments too

**Review by oz03-hub — DISMISSED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,3 +5,16 @@
...
 ### Changes
+## 6/3/2024 v1.0.1
+### Added
+- Created CHANGELOG.md, we should create one in default branch as well
+- reproduce.py and reproduce_tables.ipynb
+- These two files allow you to reproduce and analyze results from the SINR paper
+- Added instructions on how to run the files at the end of README.md, please read it before attempting to run
+### Changes
+- Changed readme
+- changed a line in viz_ica.py that had incorrect path leading to error, this should be carried to another patch branch
```
Comment (cheerstopaula) [inline CHANGELOG.md:16]: Squish CHANGELOG.md versions together

File: reproduce.py (hunk 1 above)
Comment (cheerstopaula) [reply reproduce.py]: I think that would be the best!

**Review by cheerstopaula — APPROVED** [review]
PR looks good! I would only change the reproduce.py file, and eliminate train_params from it entirely. We are not training at all in that script so it is confusing why we would be defining training parameters.

**Comment by oz03-hub** [comment]
If Angela has time we can work quickly to remove those parameters today

**Review by oz03-hub — APPROVED** [review]

**Pull request merged**